src/internal/converters/eml/eml.go

```diff
@@ -60,10 +147,11 @@ func FromJSON(ctx context.Context, body []byte) (string, error) {
 	ctx = clues.Add(ctx, "item_id", ptr.Val(data.GetId()))
 
 	email := mail.NewMSG()
-	email.AllowDuplicateAddress = true // More "correct" conversion
-	email.AddBccToHeader = true        // Don't ignore Bcc
-	email.AllowEmptyAttachments = true // Don't error on empty attachments
-	email.UseProvidedAddress = true    // Don't try to parse the email address
+	email.Encoding = mail.EncodingBase64 // Doing it to be safe for when we have eventMessage (newline issues)
+	email.AllowDuplicateAddress = true   // More "correct" conversion
+	email.AddBccToHeader = true          // Don't ignore Bcc
+	email.AllowEmptyAttachments = true   // Don't error on empty attachments
+	email.UseProvidedAddress = true      // Don't try to parse the email address
```

Comment:
Just curious, does this fix the issue we had earlier where the email didn't have an `@` or something?

Reply:
Yup, this was added upstream by us to handle that.